src/docs/introduction/upgrading-to-stencil-three.md

```diff
@@ -87,6 +87,24 @@ export const config: Config = {
 };
 ```
 
+#### `cssVarShim`
+
+`extras.cssVarShim` causes Stencil to include a polyfill for [CSS
+variables](https://developer.mozilla.org/en-US/docs/Web/CSS/--*). For Stencil
+v3.0.0 this field is renamed to `__deprecated__cssVarsShim`, and thus to retain
+the previous behavior the new option can be set in your projects
+`stencil.config.ts`:
```

Comment:
Part of this sounds a little weird to me. I think we should just copy-paste the same sentence used for `dynamicImportShim`. I don't see why we need different wording for essentially the same thing 🤷‍♂️